staff/sergiomons/js/challenges/count-word/scripts/main.js

```diff
@@ -0,0 +1,3 @@
+var count = countChart("hello world")
```

Comment:
TIP 'use strict';

Reply:
WARN ```countChart```, apart from being a wrong english word  (is not ```chart```, but ```char```), is not the correct function name. the correct function name is ```countWords```. rename it.

folder name is also incorrect. rename it to ```count-words```

Reply:
I have changed it. Thanks.